src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -182,3 +205,47 @@ const getQueryVariables = (
         : DEFAULT_PAGE_SIZE,
   };
 };
+
+const renderStatusBadge = (status): null | JSX.Element => {
```

Comment:
You moved this here in https://github.com/evergreen-ci/spruce/pull/182/files as well which I think is the one where you intended to make this change. You might want to revert the change in one of the PRs so you don't hit a merge conflict if the diffs don't end up matching